runtime/runtime-params-estimator/src/cases.rs

```diff
@@ -0,0 +1,371 @@
+use crate::stats::Measurements;
+use crate::testbed::RuntimeTestbed;
+use crate::testbed_runners::{get_account_id, measure_actions, measure_transactions, Config};
+use near_crypto::{BlsPublicKey, InMemorySigner, KeyType};
+use near_primitives::account::{AccessKey, AccessKeyPermission, FunctionCallPermission};
+use near_primitives::hash::CryptoHash;
+use near_primitives::transaction::{
+    Action, AddKeyAction, CreateAccountAction, DeleteAccountAction, DeleteKeyAction,
+    DeployContractAction, FunctionCallAction, SignedTransaction, StakeAction, TransferAction,
+};
+use rand::distributions::Alphanumeric;
+use rand::seq::SliceRandom;
+use rand::Rng;
+use std::cell::RefCell;
+use std::collections::{HashMap, HashSet};
+use std::path::PathBuf;
+
+pub fn run(config: Config) {
+    let mut m = Measurements::new();
+    // Measure the speed of processing empty receipts.
+    measure_actions("receipt only", 1, None, &mut m, &config, None, vec![], false, false);
```

Comment:
Why most of actions is measured only once?

Reply:
This number is how many times an operation is executed per transaction. It is tricky to execute many actions per single transaction, because we might be doing some optimization when there are multiple actions of the same kind inside a single transaction, also certain sequences are prohibited, e.g. we cannot create an account twice within a single transaction. Also, if create a transaction with multiple actions each of which is doing a function call the receipts produced by all function calls except the last one are going to be ingnored. So for simplicity we measure each action once per transaction, but then we have multiple transactions per block and we can figure out the overhead of how much an action costs by itself by subtracting the cost/time of processing an empty transaction.

Reply:
I see.

> also certain sequences are prohibited e.g. we cannot create an account twice within a single transaction

These basically cause only one op/txn make sense.